### PR TITLE
Toggle switch now becomes blue when ON

### PR DESCRIPTION
**Pull request by abey79**
### What

Make it clearer when toggles are ON vs. OFF with some color.

Export-1717515899832

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/6494?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/6494?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!

- [PR Build Summary](https://build.rerun.io/pr/6494)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

**Comment by Wumpf** [comment]
wouldn't it make more sense to grey out the background to the color the radio buttons have? we only use blue for selected so far

**Comment by abey79** [comment]
> wouldn't it make more sense to grey out the background to the color the radio buttons have for disabled? we only use blue for selected so far

Tried this first and it looks bad. Lacks contrast and looks disabled when off.

**Comment by emilk** [comment]
We could also try making the circle larger when on

File: crates/re_ui/src/toggle_switch.rs [hunk 1]
```diff
@@ -18,11 +18,19 @@ fn toggle_switch_ui(ui: &mut egui::Ui, height: f32, on: &mut bool) -> egui::Resp
...
-        let fg_fill = visuals.bg_fill;
-        let bg_fill = visuals.text_color();
+        let fg_fill_off = visuals.bg_fill;
+        let fg_fill_on = egui::Color32::from_rgba_premultiplied(0, 128, 255, 255);
+        let fg_fill = egui::Color32::from_rgba_premultiplied(
+            lerp_u8(fg_fill_off.r(), fg_fill_on.r(), how_on),
+            lerp_u8(fg_fill_off.g(), fg_fill_on.g(), how_on),
+            lerp_u8(fg_fill_off.b(), fg_fill_on.b(), how_on),
+            lerp_u8(fg_fill_off.a(), fg_fill_on.a(), how_on),
+        );
```
Comment (emilk) [inline crates/re_ui/src/toggle_switch.rs:28]: We should just add `lerp` member to `Color32`

Comment (abey79) [reply crates/re_ui/src/toggle_switch.rs:28]: https://github.com/emilk/egui/pull/4627

**Comment by abey79** [comment]
The blue circle is now larger. A net improvement imo.

Export-1717660021692

**Pull request merged**